### DISCUSSION TITLE
Cant access any document upload button

### DISCUSSION THREAD

**Discussion by cyberwarrior-07** [community/community#29929] [GitHub Education]
hey, i cant access any upload button in the process of uploading documents for github student developer pack. Only the camera option is shown.
![Request a discount - GitHub Education](https://user-images.githubusercontent.com/67104162/185379451-c91c7c3a-9a3a-4b67-9b10-91c99371fa51.png)

**Comment by jfullstackdev** [comment]
it's a known bug even in the past

here is the list of topics related to that

https://github.com/orgs/community/discussions?discussions_q=no+upload+button+label%3A%22GitHub+Education%22
